packages/fx-core/resource/package.nls.json

```diff
@@ -44,7 +44,7 @@
   "core.AppStudioJsonUndefined": "App Studio json is undefined",
   "core.generateArmTemplate.failNotice": "'%s' Failed to create ARM template for required Azure services.",
   "core.updateArmTemplate.failNotice": "'%s' Failed to update ARM template for required Azure services.",
-  "core.updateArmTemplate.successNotice": "The Azure resource template is updated based on your resource and capability selection. We recommend to initialize your project workspace with git init to track all changes.",
+  "core.updateArmTemplate.successNotice": "The Azure resource template is updated based on your resource and capability selection.",
```

Comment:
Does Junjie aware the change?

Reply:
@MuyangAmigo Please also help to review this pr, thx!